src/btree.c

```diff
@@ -3083,13 +3083,59 @@ btree_print_offset_table(platform_log_handle *log_handle, btree_hdr *hdr)
    platform_log(log_handle, "\n");
 }
 
+static void
+btree_print_btree_pivot_stats(platform_log_handle *log_handle,
+                              btree_pivot_stats   *pivot_stats)
+{
+   // If nothing has been updated, yet, print a msg accordingly.
+   if ((pivot_stats->num_kvs == BTREE_UNKNOWN_COUNTER)
+       && (pivot_stats->key_bytes == BTREE_UNKNOWN_COUNTER)
+       && (pivot_stats->message_bytes == BTREE_UNKNOWN_COUNTER))
+   {
+      platform_log(log_handle, "   (Pivot Stats=BTREE_UNKNOWN_COUNTER)\n");
+      return;
+   }
+
+   // Indentation is dictated by outer caller
```

Comment:
I suspect this indentation comment is dead?

Reply:
It is still needed, just to indicate why we have that leading `"   (num_kvs=%u\n"`. 

This comes about as this code was modularized out of the caller, where some indentation is needed.